File: pset6/mario/less/mario.py
from cs50 import get_int

while True:
    num = get_int("Height: ")
    if num >= 1 and num <= 8:
        break

for i in range(num):
    for j in range(num - i - 1):
        print(" ", end = "")
    for k in range(num - i - 1, num):
        print("#", end = "");
    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Enter Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = height - i - 1; k < height; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int l = 0; l < i + 1; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}